### PR TITLE
Colorize VSCode ObjectScript Output

### PR DESCRIPTION
**Pull request by a-boertien**
This PR adds functionality to colorize the VSCode ObjectScript output window, including displaying different colors for:
- Successful compilation
- Errored compilation
- Individual errors ('ERROR:', etc. prefixes)
- ObjectScript authorization error
- Classes/routines/file names in compilation status
- Quoted strings, numbers, dates/times

Examples:
colorize-output-1
colorize-output-2
colorize-output-3
colorize-output-4
colorize-output-5

### CODE REVIEW

**Comment by a-boertien** [comment]
Note this is intended as a decent starting point
Possible improvements:
- Handle other permutations of errors
- Handle other warnings/text returned by IRIS that I am not aware of
- Handle languages other than English? (-- does IRIS compilation output change based on language settings?)
- Handle ObjectScript within an error/'TEXT:' output (at the moment, only string/numeric/etc. rules apply: which highlights incorrectly in some circumstances)

**Comment by daimor** [comment]
Could you add a screenshot with the result?

**Comment by a-boertien** [comment]
Screenshots added to description

**Comment by isc-bsaviano** [comment]
I just tested the dev vsix and it adds coloring for every Output channel, not just the ObjectScript one.

**Comment by gjsjohnmurray** [comment]
@isc-bsaviano is correct. Until https://github.com/microsoft/vscode/issues/19561 gets implemented (or something like it) there's no way to set per-channel grammars.

**Comment by a-boertien** [comment]
Thanks, I was not aware. That's unfortunate and understandably not acceptable.

I will close this PR and it can be revisited once microsoft/vscode#19561 is resolved, or another workaround is found.

**Comment by gjsjohnmurray** [comment]
I submitted PR https://github.com/microsoft/vscode/pull/140876 that will allow us to use this when/if accepted.

**Comment by a-boertien** [comment]
@gjsjohnmurray I had a crack at it.. not sure if it's the best way

File: package.json [hunk 1]
```diff
@@ -451,7 +452,7 @@
         "id": "vscode-objectscript-output",
         "aliases": [],
         "mimetypes": [
```
Comment (gjsjohnmurray) [inline package.json:454]: I haven't tried this, but I think it should be OK to remove the `mimetypes` array property completely now.

**Review by gjsjohnmurray — CHANGES_REQUESTED** [review]
Looking good, but I think the `mimetypes` array property is now redundant.

**Pull request merged**